releases/net471/dotnet471-api-changes.md

```diff
@@ -0,0 +1,461 @@
+```diff
+ namespace System {
++    public static class StringNormalizationExtensions {
++        public static bool IsNormalized(this string value);
++        public static bool IsNormalized(this string value, NormalizationForm normalizationForm);
++        public static string Normalize(this string value);
++        public static string Normalize(this string value, NormalizationForm normalizationForm);
++    }
+-    public class Tuple<T1> : IComparable, IStructuralComparable, IStructuralEquatable, ITuple {
++    public class Tuple<T1> : IComparable, IStructuralComparable, IStructuralEquatable, ITuple, ITupleInternal {
```

Comment:
In general, we omit private types from syntax declarations. So for each of the Tuple and ValueTuple types, **ITupleInternal** needn't be displayed.

Reply:
It is part of the public class declaration so not sure why it has to be excluded. 

In general, APIPort export does not allow any exclusions, so any fix requires manual edits that is not desirable.